src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -519,8 +519,8 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
     <value>Problem starting the plugin '{0}'. {1}</value>
     <comment>0 - plugin path, 1 - error message </comment>
   </data>
-  <data name="PushCommandSkipDuplicateAdvertise" xml:space="preserve">
-    <value>See help for push option to automatically skip duplicates.</value>
+  <data name="PushCommandSkipDuplicateAdvertiseNuGetExe" xml:space="preserve">
+    <value>To skip already published packages, use &lt;command&gt; -SkipDuplicate</value>
```

Comment:
Thoughts on this?

```suggestion
    <value>To skip an already published packages, use the -SkipDuplicate &lt;command&gt;</value>
```

Also, is command the right term? Isn't something like `--a` called a CLI option?

Reply:
"an" is not grammatically correct in the sentence. Are you wanting to change it to a singular package for some reason?

Mine says to use your command, followed by an option. Your rewrite is not using the right term, as it's calling --a a command.

Reply:
> "an" is not grammatically correct in the sentence. Are you wanting to change it to a singular package for some reason?

Ah I misread the message 🤦‍♂️, `an ` is correct.

> Mine says to use your command, followed by an option. Your rewrite is not using the right term, as it's calling --a a command.

Ah I see. That's really confusing. Is this the pattern used by nuget.exe? I personally prefer the language `use the -SkipDuplicate option`.

Reply:
All CLI's I've ever seen use the pattern <command> <option(s)>. There's nothing unusual about that. If you want to reword it, you can propose that, but -SkipDuplicate never goes in front of the command.

Reply:
Changed `<command> -SkipDuplicate`  to `the option -SkipDuplicate`.